src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -4306,13 +4306,9 @@ void M()
             var typeInfo = model.GetTypeInfo(syntax);
             var type = typeInfo.Type;
             var conv = model.GetConversion(syntax);
-            Assert.NotNull(type);
+            Assert.Null(type);
```

Comment:
I was originally thinking we might want to try and replicate today's behavior here, but I think we'd rather be consistent with the behavior of method groups. Today, this is not meaningful, but with function pointers it is.